docs/src/main/sphinx/connector/hive.rst

```diff
@@ -375,11 +381,10 @@ Property Name                                      Description
                                                    is used for write operations. The ``${USER}`` placeholder
                                                    can be used to use a different location for each user.
 
-``hive.translate-hive-views``                      Enable translation for Hive views. (experimental)            ``false``
+``hive.translate-hive-views``                      Enable translation for :ref:`Hive views <hive-views>`.       ``false``
 
-``hive.legacy-hive-view-translation``              Use the legacy algorithm to translate Hive views. You can    ``false``
-                                                   alternatively set the ``legacy_hive_view_translation``
```

Comment:
This causes a misunderstanding that it is a session property .. when in fact it is a catalog session property

Reply:
We should list session properties directly in this table, where available.
For example we have `hive.timestamp-precision` above and i find it user-unfriendly not to mention `timestamp_precision` catalog session property next to it.

 @electrum might have  thoughts as well

Reply:
I can add it back in for the current property .. doing it for all the others is a separate, much larger task

Reply:
Arguably we could have another column in this kind of tables .. Catalo property, Catalog session property, Description, Default

But that should be consistent across all the docs.. we could do a mass update and convert all tables to use list-table and then add it..